### PR TITLE
Add examples to MSSQL module

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
Adds the folllowing examples to the module
- Query operation
- Execute operation
- Complex queries operation
- Calling stored procedures
- Batch execution

Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/1491

Related issues
- https://github.com/ballerina-platform/ballerina-standard-library/issues/1537
- https://github.com/ballerina-platform/ballerina-standard-library/issues/1539

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -15,6 +15,8 @@ rootProject.name = 'ballerina-mssql'
 include ':build-config:checkstyle'
 include ':mssql-native'
 include ':mssql-ballerina'
+include ':mssql-examples'
+
```
Comment (kalaiyarasiganeshalingam) [inline settings.gradle:19]:
```suggestion
```

**Review by daneshk — APPROVED** [review]

File: mssql-examples/batch-operation/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+* Follow one of the following ways to add MSSQL database driver JAR in the `Ballerina.toml` file:
+    * Download the JAR and update the path
+        ```
+            [[platform.java11.dependency]]
+            path = "PATH"
+        ```
+
+    * Replace the above path with a maven dependency param
+        ```
+            [platform.java11.dependency]]
```
Comment (niveathika) [inline mssql-examples/batch-operation/README.md:20]:
```suggestion
            [[platform.java11.dependency]]
```

File: mssql-examples/batch-operation/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+* Add required configurations in the `config.toml` file
+
+* Follow one of the following ways to add MSSQL database driver JAR in the `Ballerina.toml` file:
+    * Download the JAR and update the path
+        ```
+            [[platform.java11.dependency]]
+            path = "PATH"
+        ```
+
+    * Replace the above path with a maven dependency param
```
Comment (niveathika) [inline mssql-examples/batch-operation/README.md:18]:
```suggestion
    * Replace the above path with a maven dependency parameter
```

File: mssql-examples/batch-operation/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        ```
+            [platform.java11.dependency]]
+            groupId = "com.microsoft.sqlserver"
+            artifactId = "mssql-jdbc"
+            version = "9.2.0.jre11"
+        ```
+# Run the example
+
+To run the example, move into the `batch-operation` project and execute the below command.
```
Comment (niveathika) [inline mssql-examples/batch-operation/README.md:27]:
```suggestion
To run the example, move into the `batch-operation` project and execute the command below.
```

File: mssql-examples/build.gradle [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+ */
+
+apply plugin: 'java'
+
+description = 'Ballerina - MSSQL Examples'
+
+def ballerinaModulePath = "${project.rootDir}/mssql-ballerina/"
+def ballerinaDistPath = "${ballerinaModulePath}/build/target/extracted-distributions/jballerina-tools-zip/jballerina-tools-${ballerinaLangVersion}"
+def ballerinaDist = "${buildDir}/target/ballerina-distribution"
+//def examples = ["atomic-batch-operation", "batch-operation", "call-stored-procedures", "complex-queries-operation", "execute-operation", "query-operation"]
```
Comment (niveathika) [inline mssql-examples/build.gradle:28]: Any reason to comment this out?

Comment (kaneeldias) [reply mssql-examples/build.gradle]: No, that was supposed to be removed. Fixed in ad0a915

File: mssql-examples/batch-operation/README.md (hunk 2 above)
Comment (kaneeldias) [reply mssql-examples/batch-operation/README.md]: Fixed in ad0a915

File: mssql-examples/batch-operation/README.md (hunk 3 above)
Comment (kaneeldias) [reply mssql-examples/batch-operation/README.md]: Fixed in ad0a915

File: mssql-examples/batch-operation/README.md (hunk 4 above)
Comment (kaneeldias) [reply mssql-examples/batch-operation/README.md]: Fixed in ad0a915

**Comment by niveathika** [comment]
@kaneeldias There is no need to force push if you want to fix issues. We can add another commit. That is fine. Force push is not recommended since this will rewrite history and if another person is working in the same branch, their changes can also be overridden.

**Review by niveathika — APPROVED** [review]

**Pull request merged**